security/src/main/java/io/micronaut/security/rules/SecuredAnnotationRule.java

```diff
@@ -65,6 +68,16 @@ public Publisher<SecurityRuleResult> check(HttpRequest<?> request, @Nullable Rou
         if (routeMatch instanceof MethodBasedRouteMatch) {
             MethodBasedRouteMatch<?, ?> methodRoute = ((MethodBasedRouteMatch) routeMatch);
             if (methodRoute.hasAnnotation(Secured.class)) {
+
+                for (AnnotationValue<Secured> securedAnnotation: methodRoute.getAnnotationValuesByType(Secured.class)) {
+                    if (securedAnnotation instanceof EvaluatedAnnotationValue<Secured>) {
+                        Optional<SecurityRuleResult> result = securedAnnotation.booleanValue()
+                            .map(b -> (Boolean.TRUE.equals(b) ? SecurityRuleResult.ALLOWED : SecurityRuleResult.REJECTED));
+                        if (result.isPresent()) {
+                            return Mono.just(result.get());
+                        }
+                    }
+                }
                 Optional<String[]> optionalValue = methodRoute.getValue(Secured.class, String[].class);
```

Comment:
not part of this PR but this should be changed to `methodRoute.stringValues(Secured.class)` as this call is less efficient

Reply:
@graemerocher I made the change here since it made more sense as part of the 'non-repeatable' correction you mentioned above.

Reply:
I don't see these changes

Reply:
resolved via https://github.com/micronaut-projects/micronaut-security/pull/1309/commits/d9287702f262ed23433eb14f3021ed5f402512be